R/check.R

```diff
@@ -48,16 +47,20 @@ check_group <- function(obs) {
 #'
 #' @return NULL
 #'
+#' @importFrom cli cli_abort
 #' @family check
 check_group_date_unique <- function(obs) {
   group_cols <- c("reference_date", "report_date", ".group")
   obs <- coerce_dt(obs, required_cols = group_cols, copy = FALSE)
   cells <- obs[, .(count = .N), by = group_cols]
   if (any(cells[, count > 1])) {
-    stop("The input data seems to be stratified by more variables ",
-         "than specified via the `by` argument. Please provide additional ",
-         "grouping variables to `by`, ",
-         "or aggregate the observations beforehand.")
+    cli::cli_abort(
+      c(
+        "The input data seems to be stratified by more variables than",
```

Comment:
so will this print over multiple lines now or have the same behaviour as before.

Reply:
print over multiple lines; i think we should triage messaging generally as part of the be-written issue for update to style guide for error messaging.